### PR TITLE
add feature to force redirect when user already login SA-11

### PR DESCRIPTION
**Pull request by Anangga-Dipa**
TLDR:
add feature to force redirect when user already login, enable/disable using envar  FORCE_REDIRECT

Long read:
currently the flow is:
when user visit WP with additional path eg: /login/
 - if user not login it will redirect user to Login website
 - if user already login, user will stay on WP website

new flow:
when user visit WP with additional path eg: /login/ 
 - if user not login it will redirect user to Login website
 - if user already login, user will stay on WP website
 - if user already login and FORCE_REDIRECT is true, user will redirect to login page which is redirect again to PP since user already login

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ To build this plugin you need to set up following environments variable:
 - `CLIENT_REGISTER_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP register URI and redirect to it, ie: "/register-overlay/" 
 - `CLIENT_LOGIN_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP login URI and redirect to it, ie: "/login-overlay/"  
 - `CLIENT_FORGOT_PASSWORD_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP forgot password URI and redirect to it, ie: "/forgot-password-overlay/"
+- `FORCE_REDIRECT` - Optional - force redirect event if user already logged in, set 'true' for force redirect"
```
Comment (THZs) [inline README.md]: How if we change the description to be more explicit like this,
`force redirect to player portal on login button click if the player already logged in`

**Review by THZs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Anangga-Dipa) [reply README.md]: I think `on login button click` is not right statement since when user already login the button show username
this redirect is related to https://github.com/AccelByte/wordpress-react-plugin/pull/3 feature
wdyt @THZs?

Comment (THZs) [reply README.md]: Oh sorry, I thought it was in the login button.
I think we need to change the variable name to be like this `CLIENT_OVERLAY_URI_PATH_AUTO_REDIRECT` to make it more explicit and easy to understand.

- `CLIENT_OVERLAY_URI_PATH_AUTO_REDIRECT` - Optional - force redirect to player portal page if the user already logged in when accessing `CLIENT_REGISTER_URI_PATH_NAME`, `CLIENT_FORGOT_PASSWORD_URI_PATH_NAME` and `CLIENT_LOGIN_URI_PATH_NAME`, (default `true`)

File: version-customer.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "version": "1.0.0"
```
Comment (argaditya-ab) [inline version-customer.json]: I think we don't need version-customer for this repo, it should be exist only in our customer repo

**Review by argaditya-ab — APPROVED** [review]

**Review by argaditya-ab — APPROVED** [review]

File: widget/src/utils/env.ts [hunk 3]
```diff
@@ -30,3 +30,5 @@ export const registerUriPathName = process.env.CLIENT_REGISTER_URI_PATH_NAME ||
 export const loginUriPathName = process.env.CLIENT_LOGIN_URI_PATH_NAME || "";
 
 export const forgotPasswordUriPathName = process.env.CLIENT_FORGOT_PASSWORD_URI_PATH_NAME || "";
+
+export const forceRedirect = process.env.FORCE_REDIRECT || false;
```
Comment (aylwin-AB) [inline widget/src/utils/env.ts]: just checking, did you check when the value set to FORCE_REDIRECT=false? it usually become string and when you check if(forceRedirect) it will treat it as true.

usually we do something like this
export const forceRedirect = process.env.FORCE_REDIRECT==="true" || false;

but I could be wrong, could you please check if it become string or not

**Review by aylwin-AB — APPROVED** [review]

File: widget/src/utils/env.ts (hunk 3 above)
Comment (yanief) [inline widget/src/utils/env.ts]: echoing what @aylwin-AB mentioned.

since the value for `process.env.FORCE_REDIRECT` would be either `"true"` or `"false"`. Please confirm that this is this line is correct

**Review by yanief — APPROVED** [review]

File: version-customer.json (hunk 2 above)
Comment (Anangga-Dipa) [reply version-customer.json]: version customer removed and added to git ignore since it still needed when run on local to avoid `delete and add` or miss pushed

File: README.md (hunk 1 above)
Comment (Anangga-Dipa) [reply README.md]: var name updated

File: widget/src/utils/env.ts (hunk 3 above)
Comment (Anangga-Dipa) [reply widget/src/utils/env.ts]: updated and tested for no assigned, false and true value

Comment (Anangga-Dipa) [reply widget/src/utils/env.ts]: updated and tested with no assigned, false, and true value
